commons/push/integration/garmin/garmin_push_activity_details.avsc

```diff
@@ -0,0 +1,37 @@
+{
+  "namespace": "org.radarcns.push.integration.garmin",
+  "name": "GarminActivityDetails",
+  "type": "record",
+  "doc": "Fitness activity details summaries represent detailed information about discrete fitness activities, such as running or swimming, that are specifically and intentionally started by the user on their device. All wellness data, like steps and distance, contained in the activity are already represented in the Daily summary and in the corresponding Epoch summaries, so Activity Detail summaries should only be used for programs that wish to treat specific activity types in different ways, such as giving the user extra credit for going swimming three times in the same week. Activity details summaries include all data recorded by the device as part of the Fitness Activity, including GPS coordinates and all recorded sensor data. ",
+  "fields": [
+    {"name": "summaryId", "type": "string", "doc": "Unique identifier for the summary."},
+    {"name": "time", "type": "double", "doc": "Start time of the activity in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "timeReceived", "type": "double", "doc": "Time that this record was collected by a service in seconds since the Unix Epoch (s)."},
+    {"name": "startTimeOffsetInSeconds", "type": ["null", "int"], "doc": "Offset in seconds to add to time to derive the 'local' time of the device that captured the data.", "default": null},
+    {"name": "activityType", "type": ["null", "string"], "doc": "Text description of the activity type. See Appendix A for a complete list.", "default": null},
```

Comment:
There is a very long list of valid activity types. Not sure where is the best place to put it.

Reply:
In that case a URL to the activity types could work? You could give two or three examples inline, like e.g. walking, running?

Reply:
No, there is no URL. It is in the documentation provided as a PDF file. I copy and put the list as a comma-separated file in this repo and link it here?

Reply:
If it is a public PDF, we could publish it as part of this repository and link to it that way?

Reply:
Unfortunately, it is not public, it is a confidential document provided only through the dev portal available after you have access to the key

Reply:
Hmm, in that case, perhaps the entire list does need to be listed here... Alternatively, you use an enum to explicitly state what activity it is, but then if new activity types are added, the list needs to be updated to take that into account.